bip0039/src/language/mod.rs

```diff
@@ -22,7 +24,7 @@ mod spanish;
 ///
 /// The English language is always available, other languages are enabled using
 /// the compilation features.
-pub trait Language: Sized {
+pub trait Language: Sized + Ord + PartialOrd + PartialEq + Eq + Hash {
```

Comment:
why are you need these trait bounds?

Reply:
To allow `Mnemonic` to implement `Eq`. I could check equivalence on the entropy instead, so it's not a strong need. I added the remaining bounds for consistency with version 0.10, as I'm assuming `Mnemonic` is still intended to implement those traits.